### PR TITLE
HAITATON-17 Update email template johtoselvitys-valmis

### PR DESCRIPTION
**Pull request by martinkankaanranta**
# Description

Change based on Gofore maintenance ticket HAITATON-17

> Moi, https://github.com/City-of-Helsinki/haitaton-backend/blob/main/email/johtoselvitys-valmis.mjml -tiedosto pitäisi vaihtaa uuteen. Uudessa tiedostossa palautelinkin osoite ja allekirjoitus vaihdettu. Uusi tiedosto löytyy liitteenä.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [x] Other

# Instructions for testing
Please describe tests how this change or new feature can be tested.

# Checklist:

- [ ] I have written new tests (if applicable)
- [ ] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
 or other location: 

# Other relevant info
Please describe here if there is e.g. some requirements for this change or
 other info that the tester/user needs to know.

### CODE REVIEW

**Review by SatuKautt — APPROVED** [review]

File: email/johtoselvitys-valmis.mjml [hunk 1]
```diff
@@ -74,15 +78,17 @@
...
-                        href="https://response.questback.com/isa/qbv.dll/bylink?p=x6TIpQ4sCcJt9Dv1T-AuuKa-DssX27rx6er2ohTVVipooPTA8H0u-weNFWY9GsVI0">
+                        href="https://ecv.microsoft.com/zepZlZSdnT">
                     here</a>.
                 </mj-text>
                 <mj-text mj-class="basic-txt">
                     Kind regards,
                 </mj-text>
                 <mj-text mj-class="signature">
-                    Helsinki Urban Environment Resident and Business Services
+                    Helsinki city
```
Comment (umeetiusbaar) [inline email/johtoselvitys-valmis.mjml:88]: Should this be City of Helsinki? Or Helsinki City with a capital C?

Comment (AaroHuttula) [reply email/johtoselvitys-valmis.mjml:88]: @umeetiusbaar The "Urban Environment Resident and Business Services" text can be found on line 90. Line 88 is correct. Same thing can be found e.g. on lines 24-26 and 57-59.

**Comment by corvidian** [comment]
Our email building is not working properly. The actual template isn't getting updated. Looks like Gradle has changed something without us noticing it.

Can you change line 158 of `services/hanke-service/build.gradle.kts` to:
```kotlin
        from(file("${layout.buildDirectory.get()}/mjml/main/"))
```
And then run `./gradlew copyEmailTemplates` and add the changes to `services/hanke-service/src/main/resources/email/template/johtoselvitys-valmis.html.mustache` to this PR.

**Comment by corvidian** [comment]
Also, there's a text version of the email in `services/hanke-service/src/main/resources/email/template/johtoselvitys-valmis.text.mustache`, which needs to be updated separately.

**Comment by AaroHuttula** [comment]
> @AaroHuttula should I change something else other than links here: https://github.com/City-of-Helsinki/haitaton-backend/blob/HAITATON-17/update-email-johtoselvitys-valmis/services/hanke-service/src/main/resources/email/template/johtoselvitys-valmis.text.mustache

@martinkankaanranta Should the signature be changed to the updated one on this file as well?

**Review by corvidian — APPROVED** [review]

**Review by umeetiusbaar — APPROVED** [review]

**Review by AaroHuttula — APPROVED** [review]

**Comment by corvidian** [comment]
@martinkankaanranta 
1. Restore the branch or create a new branch with the same changes.
2. Open a PR for merging it to main.
3. Squash and merge the PR.
4. Azure DevOps -pipelines will build and install the changes to staging.
5. Test the changes to the emails. I added your Gofore-email to our whitelist, so you can use your own email for receiving the email.
6. Allow the DevOps-pipeline to continue to development.
7. Merge the changes from main back to dev, via a PR.

**Pull request merged**